tox.ini

```diff
@@ -16,7 +24,7 @@ commands =
 
 [testenv:docs]
 changedir = docs
-whitelist_externals =
+allowlist_externals =
```

Comment:
This will also need `poetry` to perform the `commands_pre` step, you can add it with `{[testenv]allowlist_externals}`.

Reply:
Alright, I can do that.
The tests seem to be passing without it, I don't understand why is it necessary.

Reply:
Good point. No idea but I got an error when I ran tox locally. Maybe some versions of tox need the commands in `commands_pre` to be allowed and others don't?